schemas/kits-schema.json

```diff
@@ -35,6 +35,22 @@
                 "type": "string",
                 "description": "Architecture to target"
             },
+            "environmentVariablesShellScript": {
+                "type": "string",
+                "description": "Filename of a shell script which sets environment variables for the kit"
+            },
+            "environmentVariablesShellScriptDirectory": {
```

Comment:
I prefer to add as few properties as possible to make this work.  Can you achieve your goal without this?  Ideally the scripts are able to run regardless of the current working directory (e.g. they patch them up themselves).

Reply:
Yes, no problem